dts/bindings/rtc/microcrystal,rv3028.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 ANITRA system s.r.o.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Micro Crystal RV3028 RTC
+
+compatible: "microcrystal,rv3028"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+  clkout-frequency:
+    type: int
+    enum:
+      - 32768
+      - 8192
+      - 1024
+      - 64
+      - 32
+      - 1
+    description: |
+      Frequency of the CLKOUT signal in Hertz (Hz). When disabled, the CLKOUT pin is LOW.
```

Comment:
what does "disabled" mean here? I think it should be clarified

Reply:
I'll update the description to say: `If omitted, the CLKOUT pin is LOW.` Is it ok?